content/en/docs/armory-admin/kubernetes-account-add.md

```diff
@@ -21,7 +21,7 @@ At a high level, Spinnaker operates in the following way when deploying to Kuber
 * Each Kubernetes account is configured in the `SpinnakerService` manifest under `spec.spinnakerConfig.config.providers.kubernetes.accounts` key if using the Operator, or under the master `.hal/config` YAML file (also known as a "Halconfig"), as an entry in the deploymentConfiguration's `providers.kubernetes.accounts` array if using Halyard.  Each entity has these (and other) fields:
   * `name`: A Spinnaker-internal name used to refer to the Kubernetes account.  This is the item you will select in various Kubernetes stages to indicate that you would like to deploy to this particular Kubernetes account (which again, consists of a Kubernetes cluster/credential/context set)
   * `kubeconfigFile`: A file path referencing the contents of the kubeconfig file for connecting to the target cluster. When using the Operator, this can be any name that should match an entry in `spec.spinnakerConfig.files` where the file contents is copied. If using Halyard, this is a physical file path. This field supports referencing files stored in external [secret engines]({{< ref "secrets" >}})
-  * `onlySpinnakerManaged`: A boolean that indicates whether Spinnaker should only manage / display entities that were created by Spinnaker (i.e., those that have the Spinnaker labels and annotations that indicate that they were created by Spinnaker)
+  * `onlySpinnakerManaged`: When `true`, Spinnaker only caches and displays applications that have been created by Spinnaker.  Before placing in a `false` state, you should review the Kubernetes cluster configuration.  When `false`, Spinnaker analyzes the cluster and automatically attempts to configure and populate applications for resources already present in Kubernetes, unless limited with `omitNamespaces`. You should note the [increased possibilities of zombie applications in MySQL deployments](https://kb.armory.io/s/article/Autogenerated-Applications-Cannot-be-Deleted-w-onlySpinnakerManaged-set-to-false).
```

Comment:
waiting for Daniel Williams to review

Reply:
This looks good, however the zombie applications aren't unique to MySQL.  They happen with Redis too.  MySQL just has a second issue where the cache doesn't clean up, so the zombie apps don't go away when the setting is set back to true.  Also, might be a good idea to use "auto-generated application placeholders" or similar instead of "zombie applications".